SwrveSDK/src/main/java/com/swrve/sdk/localstorage/SQLiteLocalStorage.java

```diff
@@ -260,6 +264,8 @@ public void setMultipleCacheEntries(List<Entry<String, Entry<String, String>>> c
 
     @Override
     public void close() {
+        SwrveLogger.i("SQLLite", "we're closing database isDbLockedByCurrentThread: " + database.isDbLockedByCurrentThread() + " isOpen: " + database.isOpen());
```

Comment:
Same as above, I see these guys as temp that will be removed after we get a more reasonable console from Jenkins.